### PR TITLE
Timestamp

### PR DESCRIPTION
**Pull request by s-aderhold**
modified reading of data to accommodate new line for timestamp in the header

### CODE REVIEW

File: FFt_math.py [hunk 1]
```diff
@@ -51,40 +53,39 @@ def parseCavDat(read_data):
...
-#print(cavDat3)
-#    print('cavDat1[0:5]')
-#    print(cavDat1[0:5])
-#    print('cavDat2[0:5]')
-#    print(cavDat2[0:5])
-#    print('cavDat3[0:5]')
-#    print(cavDat3[0:5])
-
-    return([cavDat1,cavDat2,cavDat3,cavDat4])
+    # print(cavDat3)
```
Comment (lisazacarias) [inline FFt_math.py:56]: Can probably delete these lol

File: FFt_math.py [hunk 2]
```diff
@@ -124,8 +125,8 @@ def compatibleMkdirs(filename):
...
     'ACCL:L3B:34',
     'ACCL:L3B:35',
-    ]
+]
 
-#getter
-def CM_IDs(): return CRYOMODULE_IDS
 
+# getter
+def CM_IDs(): return CRYOMODULE_IDS
```
Comment (lisazacarias) [inline FFt_math.py:132]: Do we need a getter?

**Review by lisazacarias — APPROVED** [review]

File: FFt_math.py (hunk 2 above)
Comment (s-aderhold) [reply FFt_math.py:132]: don't think so

File: FFt_math.py [hunk 3]
```diff
@@ -51,40 +53,39 @@ def parseCavDat(read_data):
...
+    return
 
 
 def compatibleMkdirs(filename):
     makedirs(path.dirname(filename), exist_ok=True)
     return (filename)
 
+
 # CMID = <SC linac> : <CM ID>
 CRYOMODULE_IDS = [
```
[the remark this answers is not in the document]
Comment (s-aderhold) [reply FFt_math.py:90]: yup

**Pull request merged**